tmexp/preprocess.py

```diff
@@ -128,15 +116,15 @@ def preprocess(
             lang_count[lang] += 1
             seen_files.add((file_path, blob_hash))
         files_info[ref][file_path] = {"blob_hash": blob_hash, "language": lang}
-    logger.info("Found {} parsable files:".format(raw_count))
+    logger.info("Found %d parsable files:" % raw_count)
```

Comment:
for logger entries, you don't need to use `%`, you can just use `,`. Here it's not that useful but when the formatstring takes several arguments it helps, like several cases below (I didn't add a comment for each of them).

Reply:
oke, will use that convention instead when there is more then one argument